app/src/popup/subcomponents/ShrinkedView.tsx

```diff
@@ -47,17 +54,37 @@ export const ShrinkedView = () => {
 
     return (
         <>
-            <DurationHeader />
-            <div className="shrinked-view-container">
-                <div className="shrinked-view-icon-container">
-                    <img className="shrinked-view-icon" width={icon.size} height={icon.size} src={icon.src} />
-                </div>
-                <div className="shrinked-view-time-spent-text-container">
-                    <div className="shrinked-view-time-spent-text">
-                        {getHours(timeInSeconds)}:{getMinutes(timeInSeconds)}:{getSeconds(timeInSeconds)}
+            {!detailsScreenWebsite ? <DurationHeader /> : null}
+            {detailsScreenWebsite ? (
+                <DetailsView
+                    website={detailsScreenWebsite}
+                    onBackButtonClick={() => {
+                        setDetailsScreenWebsite(null);
+                        setDetailsScreen(false);
+                    }}
+                />
+            ) : (
+                <div className="shrinked-view-container">

```

Comment:
Whole `ShrinkedView` looks more and more like a tile of `ExtendedView`. Maybe there will be a time to make a shared component out of it.